src/DataContainer/CalendarEvents.php

```diff
@@ -1165,7 +1165,7 @@ public function showFieldValue(DataContainer $dc): string
             }
 
             if (!empty($help)) {
-                $help = '<p class="tl_help tl_tip">'.$help.'</p>';
+                $help = '<p class="tl_help tl_tip tl_full_height">'.$help.'</p>';
```

Comment:
@markocupic Thank you for merging 😃 
It seems that this change in the function `showFieldValue()` does not help for the longer event descriptions, because the additional class is not generated in the HTML output in backend. 
So, we can not easily improve it...

PS: If I add `tl_full_height` manually in my browser (debugging view), it would work.

Reply:
> the additional class is not generated in the HTML output in backend.

@jonasmueller1 Are you sure?

<img width="308" alt="image" src="https://user-images.githubusercontent.com/1525166/183260610-e068542e-c175-4c4c-b4ab-28ea60e6e308.png">

May be I'm missunderstanding you ;-)

Reply:
@markocupic 
Ah, it is only generated for one field `Event-Alias` for tours (in green), but NOT for all others (in red):

![grafik](https://user-images.githubusercontent.com/99007003/183261000-8ebff5f7-5ec6-44fd-8b98-760ce09c8f5a.png)

I checked the full output of our test tour.